messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -183,32 +185,17 @@ protected PooledStreamingEventProcessor(PooledStreamingEventProcessor.Builder bu
                                       .claimExtensionThreshold(claimExtensionThreshold)
                                       .clock(clock)
                                       .maxClaimedSegments(maxClaimedSegments)
+                                      .tokenStoreInitializer(new TokenStoreInitializer())
```

Comment:
I'm wondering whether we should provide the runnable, or if we should share the `initialSegmentCount` and `intialToken` with the `Coordinator.

Reply:
Changed this.